src/internal/m365/collection/exchange/contacts_container_cache.go

```diff
@@ -104,8 +107,10 @@ func (cfc *contactContainerCache) Populate(
 		cfc.userID,
 		baseID,
 		false)
+	ctx = clues.Add(ctx, "num_enumerated_containers", len(containers))
```

Comment:
minor: I'd love to see this go into a `count.Bus`, but I don't know how much investment that would take.  nbd.

Reply:
I think that is a good goal to have. I think it would be best done in a separate PR though since that's not directly related to this logging and the bus isn't present in the resolver already. Mostly I don't want this PR to snowball into something larger since it should be safe to merge by itself